auth/user_mgt_test.go

```diff
@@ -995,7 +995,7 @@ func TestInvalidDeleteUser(t *testing.T) {
 }
 
 func TestMakeExportedUser(t *testing.T) {
-	rur := &userQueryResponse{
+	queryResponse := &userQueryResponse{
```

Comment:
I couldn't figure out what `rur` stood for, so I renamed it. (I wanted my test to be consistent with this one.) Happy to change it back if someone can enlighten me.